### ISSUE TITLE
Invalid entity-per-chunk-save-limit config

### ISSUE THREAD

**Issue by andris155** [YouHaveTrouble/minecraft-optimization#49]
Paper writes this error.
```
[10:01:25 ERROR]: Invalid entity-per-chunk-save-limit config, 'ender_signal' is not a valid entity type. Correct this in paper.yml.
[10:01:25 ERROR]: Invalid entity-per-chunk-save-limit config, 'firework' is not a valid entity type. Correct this in paper.yml.
[10:01:25 ERROR]: Invalid entity-per-chunk-save-limit config, 'splash_potion' is not a valid entity type. Correct this in paper.yml.
[10:01:25 ERROR]: Invalid entity-per-chunk-save-limit config, 'thrown_exp_bottle' is not a valid entity type. Correct this in paper.yml.
```

**Comment by YouHaveTrouble** [comment]
Corrected in 3e5f77b.
Apparently bukkit's EntityType enum does not reflect mojang entity IDs, so there's that.
Correct IDs can be found on minecraft wiki of all places.

**Comment by andris155** [comment]
Thanks!
